doc/develop/flash_debug/host-tools.rst

```diff
@@ -200,6 +200,43 @@ Windows PATH. A specific bossac executable can be used by passing the
 
    WSL is not currently supported.
 
+LinkServer Debug  Host Tools
+****************************
+
+Linkserver is a utility for launching and managing GDB servers for NXP debug probes, which also provides a command-line target flash programming capabilities. Linkserver can be used with NXP MCUXpresso for Visual Studio Code implementation, with custom debug configurations based on GNU tools or as part of a headless solution for continuous integration and test. Linkserver can be used with MCU-Link, LPC-Link2, LPC11U35-based and OpenSDA based standalone or on-board debug probes from NXP. The Linkserver installer also includes the firmware update utilities for MCU-Link and the LPCScrypt utility for use with LPC-Link2. Linkserver can also be installed using the MCUXpresso Installer.
```

Comment:
Zephyr documentation style is to fill the paragraph by adding newlines, rather than having very long lines in a text file. (This applies to the entire diff in this PR)